modules/ROOT/pages/Development/Satisfactory/ConveyorRendering.adoc

```diff
@@ -1,5 +1,10 @@
 = Conveyor Rendering
 
+[NOTE]
+====
+Ben (CSS employee) has confirmed that this system will be changing in a future update, and that these steps will soon change, or may no longer be required. Consider waiting to set up these conveyor materials until the new system is released.
+====
+
```

Comment:
Does this notice, which I added in a commit recently, sound good?

Reply:
Is it worth it to tell them to wait? We don't really know when that update will come right?

Reply:
[This message](https://discord.com/channels/555424930502541343/862002311579435028/864921516160581633) (#blueprints-help) from Ben makes it sound like it will be coming in U5.